web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -33,9 +35,13 @@
 from accelerator_abstract.models.base_user_utils import is_employee
 User = get_user_model()
 
+today = datetime.today()
+DAYS_IN_MONTH = calendar.monthrange(today.year, today.month)[1]
```

Comment:
This assumes that we're only interested in the current month. Users who consult this page in February to find out about office hours in March might find themselves disappointed! :)

I would suggest that we just say there's 31 days in a month, and let the front end discard anything that it doesn't want to show. (just as we are already padding out the week)